### ISSUE TITLE
Pytanie o zmiany API WZA wprowadzane 23 maja

### ISSUE THREAD

**Issue by Maciej-Wilkosz** [allegro/allegro-api#9025]
Dzień Dobry, na podstawie zmian wymienionych w : https://help.allegro.com/sell/pl/a/wysylam-z-allegro-sprawdz-jak-nadawac-przesylki-allegro-inpost-aby-je-prawidlowo-rozliczyc-m0yO60k3oFK?utm_source=coma_email&utm_medium=0fd4be62-7ff6-47dd-a3e2-4e43b1d1518f#sprawdz-liste - jest wskazane jakoby teraz przy tworzeniu wysyłki do pola id metody dostawy trzeba będzie do samej metody dostawy doklejać id umowy własnej (np 2488f7b7-5d1c-4d65-b85c-4cbcf253fd93#id_umowy_wlasnej). Czy tak mamy tą zmianę interpretować czy jednak żadne zmiany implementacyjne nie będą wymagane?

**Comment by MaciejFrackowiak** [comment]
Dokładnie tak, musisz przekazać identyfikator metody dostawy + id umowy własnej dla wskazanej metody dostawy.

**Comment by CSVK2022** [comment]
@MaciejFrackowiak nie wiem czy dobrze rozumiemy:
rozliczając Allegro Paczkomaty Inpost z Allegro (nie z InPost) dalej wymagane jest dodanie id umowy własnej?

**Comment by MaciejFrackowiak** [comment]
@CSVK2022 Tak, i idąc dalej przekazanie jej id w requeście w polu odpowiedzialnym za daną metodę dostawy, np. "2488f7b7-5d1c-4d65-b85c-4cbcf253fd93#id_umowy_wlasnej" dla metod Allegro Paczkomaty InPost.

**Comment by kulisz** [comment]
@MaciejFrackowiak podepnę się pod wątek - rozumiem że w odpowiedzi z /order/checkout-forms/{id} w sekcji delivery pozostaje po staremu czyli `"delivery": {
"address": {},
"method": {
"id": "1fa56f79-4b6a-4821-a6f2-ca9c16d5c925",
"name": "Allegro Paczkomaty InPost"
},
"pickupPoint": {},
"cost": {},
"time": {},
"smart": true,
"calculatedNumberOfPackages": 1
},`, tak?

**Comment by MaciejFrackowiak** [comment]
@kulisz W obsłudze zamówień nic się nie zmienia - w delivery.method.id zwrócimy wybraną przez kupującego metodę dostawy. Twój przykład jednak jest niespójny - podane id metody to "Dostawa i jej koszt ustalane ze sprzedającym", a nie "Allegro Paczkomaty InPost" - tak dla pełnej zgodności.
